lang/python/python-requests-oauthlib/Makefile

```diff
@@ -1,6 +1,4 @@
 #
-# Copyright (C) 2007-2016 OpenWrt.org
```

Comment:
Looking at the [package history](https://github.com/openwrt/packages/commits/master/lang/python/python-requests-oauthlib), I think you can assign copyright to yourself :smile: